modules/png-extraction/ImageExtractor.py

```diff
@@ -16,7 +16,7 @@
 import numpy as np
 import pandas as pd
 import pydicom as dicom 
-import png
+# import png
```

Comment:
why is this commented? Is png module unnecessary for png-extraction?

Reply:
Oh. That's not expected. I made the change while testing. I will update the PR by reverting the comment.